core/templates/dev/head/editor/ExplorationFeedback.js

```diff
@@ -161,23 +236,48 @@ oppia.controller('ExplorationFeedback', [
     {id: 'not_actionable', text: 'Not Actionable'}
   ];
 
-  $scope.getHumanReadableStatus = function(status) {
-    for (var i = 0; i < $scope.STATUS_CHOICES.length; i++) {
-      if ($scope.STATUS_CHOICES[i].id === status) {
-        return $scope.STATUS_CHOICES[i].text;
+  return {
+    STATUS_CHOICES: angular.copy(_STATUS_CHOICES),
+    getLabelClass: function(status) {
```

Comment:
I think it is a bit uglier, but it's also more angular-esque. It's almost an antipattern to perform any markup or styling logic in JS when we use Angular nearly everywhere else in the codebase. It's also a lot longer to perform it in JavaScript.

Re: testing it, I agree that it's a bit harder to test the markup, but testing the the JS by itself is almost moot. It's not really testing whether the elements are correct, it's just testing whether the right styling is used based on the calls. This means the callsites (within HTML in this case) need to still be verified to ensure they are using those functions correctly. Testing the styling using integration tests would probably be an adequate way to test something like this, since it's styling anyway.

I'm just opposed to performing any sort of styling in JavaScript since our JS files are used for logic nearly everywhere else. Feel free to make an executive decision here, though.